File: module_wanandroid/src/main/java/com/yyxnb/module_wanandroid/ui/WanWebActivity.java
package com.yyxnb.module_wanandroid.ui;

import android.os.Bundle;

import androidx.databinding.DataBindingUtil;

import com.yyxnb.lib_arch.annotations.BindRes;
import com.yyxnb.lib_arch.base.IActivity;
import com.yyxnb.lib_view.titlebar.TitleBar;
import com.yyxnb.lib_webview.WebActivity;
import com.yyxnb.module_wanandroid.R;
import com.yyxnb.module_wanandroid.databinding.ActivityWanWebBinding;

@BindRes
public class WanWebActivity extends WebActivity implements IActivity {

    private ActivityWanWebBinding binding;
    private TitleBar mTitle;
    private String title;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        binding = DataBindingUtil.setContentView(this, R.layout.activity_wan_web);
        super.onCreate(savedInstanceState);
    }

    @Override
    public void initView(Bundle savedInstanceState) {
        mTitle = binding.iTitle.vTitle;

        title = getIntent().getStringExtra("title");
        url = getIntent().getStringExtra("url");

        mTitle.getCenterTextView().setText(title);
        mTitle.setBackListener(v -> finish());

        mLinearLayout = findViewById(R.id.ll_content);


        initAgent(url, mLinearLayout);
    }
}

File: module_wanandroid/src/main/java/com/yyxnb/module_wanandroid/ui/WanWebActivity.kt
package com.yyxnb.module_wanandroid.ui

import android.os.Bundle
import android.view.View
import androidx.databinding.DataBindingUtil
import com.yyxnb.arch.annotations.BindRes
import com.yyxnb.arch.base.IActivity
import com.yyxnb.module_wanandroid.R
import com.yyxnb.module_wanandroid.databinding.ActivityWanWebBinding
import com.yyxnb.view.titlebar.TitleBar
import com.yyxnb.webview.WebActivity

@BindRes
class WanWebActivity : WebActivity(), IActivity {

    private var binding: ActivityWanWebBinding? = null
    private var title: String? = null
    private var mTitle: TitleBar? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        binding = DataBindingUtil.setContentView(this, R.layout.activity_wan_web)
        super.onCreate(savedInstanceState)

    }

    override fun initView(savedInstanceState: Bundle?) {
        mTitle = binding!!.iTitle.mTitle
        mLinearLayout = binding!!.mLinearLayout;

        title = intent.getStringExtra("title")
        url = intent.getStringExtra("url")
        mTitle?.centerTextView?.text = title
        mTitle!!.setBackListener { finish() }

        initAgent(url, mLinearLayout)
    }
}